### DISCUSSION TITLE
Does mpdf support page-break-before inside tables?

### DISCUSSION THREAD

**Discussion by laimermic** [mpdf/mpdf#1618] [Q&A] [answered]
I would like to break the page before a certain row in a table using page-break-before: always
But somehow it doesn't work...

**Comment by finwe** [comment, marked as answer]
No.

https://mpdf.github.io/tables/tables.html
> Block-level tags (DIV, P etc) are ignored inside tables, including any CSS styles

**Reply by finwe** [reply]
Best course of action would be starting a new table to create a page break, with a manual `<pagebreak>` if necessary.

**Reply by laimermic** [reply]
Ok, thank you
